core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -155,6 +171,27 @@ public synchronized ConfigCheckReport getConfigCheckReport() {
     return mConfigCheckReport;
   }
 
+  /**
+   * Logs the configuration check report information.
+   */
+  public synchronized void logConfigReport() {
```

Comment:
is there an easy way for the user to tell whether this report is for masters or workers?

Reply:
For the report, we can achieve this by traversing all the inconsistent properties and get their scopes.
SERVER: contains both master and worker properties or contains any server properties.
MASTER: only contains master properties.
WORKER: only contains worker properties.

Reply:
right now we print the full report in a very condensed format that might be hard for users to understand. What do you think of only printing the first 5 or so errors/warnings in a very human-readable format, then pointing the user to the UI/CLI if they want to get the full list?

Reply:
I agreed to only print a limited set of errors/warnings if there are many and direct the users to UI/CLI. For the human-readable format, I prefer the maintain the format here and if users want human-readable format, they can visit the web UI/CLI. 

The info contains too much layer and it would be best to show them as a table(or with indentation) like what we did in the web UI/CLI which looks very strange in our log.

Reply:
sgtm